### DISCUSSION TITLE
Needed a feature the makes my task execute on a dedicated worker client

### DISCUSSION THREAD

**Discussion by missioncoding** [Netflix/conductor#3647] [Ideas]
I am looking for a feature where i want to make sure my task will be executed by a particular client.
If there is already a feature for this i would be happy to know.

I am having some tasks that need to be performed on a private network which only some conductor worker clients running will have access.

If i submit a workflow/task currently i see any available client will start executing which may result in failure. hence i want to my task to be executed by some dedicated clients

**Comment by manan164** [comment]
Hi @missioncoding , you can use taskToDomain in case you want the task to be executed by the particular worker. Please check more here :- https://orkes.io/content/developer-guides/task-to-domain

**Comment by missioncoding** [comment]
Hello @manan164 thanks for the reply.

Yes. i was looking at isolation groups and task domains documentation. The task domains seem to be one but i never used them so far. i don't see a clear documentation with an example is available.  Can you please provide some support on this. Also my task can be executed on any worker if its a public ip input.. In case the worker input is having a private ip i need to take a call to redirect to a dedicated worker client.

**Comment by missioncoding** [comment]
Hi @manan164 thanks the task domain worked.
